src/sayt/sayt.ts

```diff
@@ -0,0 +1,20 @@
+/** The name of event fired when SAYT should be hidden. */
```

Comment:
Should it be 'The name of THE event fired when SAYT should be hidden.' to be consistent with the wording in autocomplete?

Reply:
Updated.